tests/drivers/uart/uart_mix_fifo_poll/testcase.yaml

```diff
@@ -0,0 +1,73 @@
+common:
+  tags:
+    - drivers
+    - uart
+    - ci_tests_drivers_uart
+  depends_on: gpio
+  harness: ztest
+  platform_allow:
+    - nrf54l09pdk/nrf54l09/cpuapp
+  integration_platforms:
+    - nrf54l09pdk/nrf54l09/cpuapp
+  harness_config:
+    fixture: uart_loopback
+tests:
+  nrf.extended.drivers.uart.uart_mix_poll:
+    extra_configs:
+      - CONFIG_UART_INTERRUPT_DRIVEN=n
+      - CONFIG_UART_ASYNC_API=n
+      - CONFIG_UART_0_ENHANCED_POLL_OUT=n
```

Comment:
Any option that has `UART_0` can be removed (also in all configurations bellow).